tutorials/source-modeling/plot_beamformer_lcmv.py

```diff
@@ -0,0 +1,169 @@
+"""
+Source reconstruction using an LCMV beamformer
+==============================================
+
+This tutorial gives an overview of the beamformer method
+and shows how to use an LCMV beamformer to reconstruct source
+activity.
+
+.. contents:: Page contents
+   :local:
+   :depth: 2
+
+A beamformer is a spatial filter that reconstructs source activity by scanning
```

Comment:
Consider adding a section heading here "background on beamformers" or "introduction to beamformers".  I would add to that section a brief discussion of what depth bias is and why it occurs, and mention that there are ways of dealing with it (described below).

Reply:
Attempted this in now